src/actions.rs

```diff
@@ -376,6 +381,15 @@ impl Action {
             .set_section_info(change_elder.new_section);
     }
 
+    pub fn get_section_split_votes(&self) -> Vec<ParsecVote> {
+        // The section name is currently just a signed number, so we just pick an arbirary rule to
+        // generate two new section names.
+        let our_section_name = (self.our_section().0).0;
+        (our_section_name + 1..=our_section_name + 2)
```

Comment:
I had the hardest time parsing that `..`
what about:
1..2.map(|name_offset| ParsecVote::NewSectionInfo(SectionInfo(Section(our_section_name), 0)))

Reply:
`1..2` yields only `1`. You would need `1..=2`. Or maybe cleaner: `[1, 2].into_iter()`

Reply:
Yep, whichever works :-)

Reply:
can also be 1..3, which probably is the simplest